documentation/API-linting-Implementation-Guideline.md

```diff
@@ -0,0 +1,76 @@
+# CAMARA OpenAPI Linting Rules Implementaion Guideline [ How to integrate the rules into CAMARA repository ]
+
+## Introduction
+
+This guide provides instructions on implement linting rules for the CAMARA API using two methods: <b>Spectral Linting</b> and <b>Megalinter with Spectral Linting.</B>
```

Comment:
This guide provides instructions to implement linting rules for the CAMARA APIs

Reply:
Updated